# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/credentialmanagement/identity/tests/browser/browser_single_concurrent_identity_request.js
# Commit: ccd96b99aea2
# Full Hash: ccd96b99aea28e8214e585b864d3fb2245bac142
# Author: Atila Butkovits <abutkovits@mozilla.com>
# Date: 2025-06-07 08:31:23
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Revert "Bug 1969015, 1969017: apply code formatting via Lando" for causing failures complaining about fedcm-no-registered-idps.https.html.
#   
#   This reverts commit ba33457e0cbac24a8b7d963c45138c62e92a74ae.
#   
#   Revert "Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz"
# ==============================================================================

diff -r 4bfadfc33207 -r ccd96b99aea2 dom/credentialmanagement/identity/tests/browser/browser_single_concurrent_identity_request.js
--- a/dom/credentialmanagement/identity/tests/browser/browser_single_concurrent_identity_request.js	Fri Jun 06 21:07:41 2025 +0300
+++ b/dom/credentialmanagement/identity/tests/browser/browser_single_concurrent_identity_request.js	Fri Jun 06 21:35:18 2025 +0300
@@ -44,8 +44,8 @@
   ok(concurrentResponse.name, "expect a DOMException which must have a name.");
   is(
     concurrentResponse.name,
-    "NotAllowedError",
-    "Expected 'NotAllowedError', but got '" + concurrentResponse.name + "'"
+    "InvalidStateError",
+    "Expected 'InvalidStateError', but got '" + concurrentResponse.name + "'"
   );
 
   // Close tabs.